roles/vmware-ova/ovfconf/files/ovfconf.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Configure initial address based on OVF environment
+After=vmtoolsd.service syslog.target
+Wants=network-pre.target
+Before=network-pre.target vyatta-router.service
+ConditionVirtualization=vmware
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/local/sbin/ovfconf
```

Comment:
The script should not be outside of the image.
I've made a prototype of a config modification script: https://github.com/vyos/vyos-1x/blob/current/src/utils/initial-setup
Common functionality can be moved to a module, and then we can produce two scripts, one for OVF first boot, and another for interactive installation.

Reply:
This would require changes to the `vyos-1.x`-repository, right?